maintainers/maintainer-list.nix

```diff
@@ -2587,6 +2587,13 @@
     githubId = 46527539;
     name = "ChaosAttractor";
   };
+  chayleaf = {
```

Comment:
When updating this file to solve the merge conflict notice that we also now have a ci check to ensure the file is kept alphabetical sorted.